distribution/changelog.txt

```diff
@@ -7,6 +7,7 @@
 - Fix: [#18122] Ghosts count towards “Great scenery!” guest thought.
 - Fix: [#18134] Underground on-ride photo section partially clips through adjacent terrain edge.
 - Fix: [#18257] Guests ‘waiting’ on extended railway crossings.
+- Fix: [#18287] Missing validation on invalid characters in file name.
```

Comment:
#17763 does sound like the same problem.

```suggestion
- Fix: [#17763] Missing validation on invalid characters in file name.
```

Reply:
It does indeed.